src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BitcoinActivationRule.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using Microsoft.Extensions.Logging;
 using NBitcoin;
 using Stratis.Bitcoin.Consensus.Rules;
+using Stratis.Bitcoin.Networks;
```

Comment:
Is this needed?

Reply:
Yeah. On line 9 we have xmldoc: 
```
<see cref="BitcoinMain"
```

It marked as unresolved by resharper without that reference.